tests/middleware/test_compression_middleware.py

```diff
@@ -275,6 +275,6 @@ def test_brotli_middleware_invalid_mode() -> None:
 
 def test_invalid_compression_middleware() -> None:
     try:
-        create_test_client(route_handlers=[handler], compression_config=CompressionConfig(backend="super-zip"))  # type: ignore
+        create_test_client(route_handlers=[handler], compression_config=CompressionConfig(backend="super-zip"))
```

Comment:
There are two tests for loading from a string that both actually load using the enum:

1. https://github.com/starlite-api/starlite/blob/a160eed488c2f7d30c7b57077c9d2086a14c9625/tests/middleware/test_compression_middleware.py#L97
2. https://github.com/starlite-api/starlite/blob/a160eed488c2f7d30c7b57077c9d2086a14c9625/tests/middleware/test_compression_middleware.py#L136

Reply:
Good catch.  I thought I had written these in the original release but I didn't examine the test cases in enough detail.  Updated